web/src/widget_modal.ts

```diff
@@ -68,3 +80,39 @@ export function frame_poll_message_content(): string {
         .filter(Boolean);
     return "/poll " + question + "\n" + options.join("\n");
 }
+
+export function todo_list_tasks_setup(): void {
+    const $todo_options_list = $("#add-todo-form .todo-options-list");
+    $todo_options_list.on("input", "input.todo-input", (event) => {
+        add_option_row(event, "TODO");
+    });
+    $todo_options_list.on("input", "input.todo-description-input", (event) => {
+        add_option_row(event, "TODO");
+    });
```

Comment:
I don't think we need to add a row on detecting input in the description field. The task title should be enough